.github/workflows/ci.yml

```
@@ -21,6 +21,13 @@ jobs:
           - '8.56'
           - '8'
           - '9'
+
+        include:
+          - executeLint: true
+            node: 20
+          - executeCoverage: true
+            node: 20
```

I think is enough to run coverage once, if yes we might limited its execution to `node: 20` on `os: ubuntu-latest`:
```suggestion
          - executeCoverage: true
            node: 20
            os: ubuntu-latest
```

---

> I think is enough to run coverage once

Although this is enough for `eslint-plugin-import-x`, it is generally not a good practice in case there are some platform-specific codes that need to be tested.

Lint only needs to run on one os though.

---

>  Although this is enough for eslint-plugin-import-x, it is generally not a good practice in case there are some platform-specific codes that need to be tested.

I was going for performance and didn't think about coverage on platform specific code.
I removed the `matrix.executeCoverage` value. 

> Lint only needs to run on one os though.

Done